bundles/cis-k8s-1.5.1/NOTICE

```diff
@@ -0,0 +1,11 @@
+The following notice applies to these files originally from the Gatekeeper Library (https://github.com/open-policy-agent/gatekeeper-library)
+5.2.1_psp-privileged-container.yaml
+5.2.2-5.2.3_psp-host-namespace.yaml
+5.2.4_psp-host-network-ports.yaml
+5.2.5_psp-allow-privilege-escalation-container.yaml
+5.2.6_psp-pods-allowed-user-ranges.yaml
```

Comment:
I don't see this constraint anywhere in here?

That this sort of drift exists in an initial PR is a strong indicator that we should automate this somehow.

Reply:
I had the wrong filename in there. Thanks for catching.